apps/finance/app/src/components/TransferRow.js

```diff
@@ -61,12 +60,12 @@ class TransferRow extends React.Component {
       true,
       { rounding: 5 }
     )
-    const formattedDate = formatHtmlDatetime(date)
+    const formattedDate = format(date, "yyyy-MM-dd'T'HH:mm:ss.SSSxxx")
```

Comment:
Note that these can all be replaced by [`formatHtmlDatetime()` once that's updated](https://github.com/aragon/aragon-ui/pull/223).